### DISCUSSION TITLE
Static Site Generator

### DISCUSSION THREAD

**Discussion by JahanzaibTayyab** [vercel/next.js#44918] [Help]
I want to generate a static site generation with next.js 13, how i can generate it?  , I used the build command for this.

my script in package.json 

```
"scripts": {
    "dev": "next dev",
    "build": "next build && next export",
    "start": "next start",
    "lint": "next lint"
  },
```
 but got an error every time ,
```
 PageNotFoundError: Cannot find module for page: /
    at Object.getPagePath 
```
this command works fine with next12

**Comment by tirogen** [comment]
https://beta.nextjs.org/docs/app-directory-roadmap#planned-features

next export is still work in progress
